### DISCUSSION TITLE
Use waybar for title bars

### DISCUSSION THREAD

**Discussion by aylurus** [WillPower3309/swayfx#256] [Ideas]
Instead of using regular title bars, every window could have its own instance of Waybar displayed above it. This would maximize configurability of title bars and automatically enable support for icons. The Waybar instance would need to be given its position and length. The original title bar could simply be made invisible by settings its alpha to zero. Then, if it is rendered above the Waybar instance, the original mouse interactions would be preserved. It's probably more complicated, but I wanted to share the idea and hear what others think about it.

**Comment by burein-ita** [comment]
\> It's probably more complicated
\> yes
